components/automate-ui/src/app/page-components/job-schedule-form/job-schedule-form.component.html

```diff
@@ -38,12 +38,12 @@ <h4>Set optional scan date and schedule for scans</h4>
         <div class="field">
           <chef-form-field formGroupName="datetime">
             <div class="datetime">
-              <select formControlName="month">
-                <option *ngFor="let n of months" [value]="n">{{ n | number:'2.0-0' }}</option>
-              </select>
               <select formControlName="date">
                 <option *ngFor="let n of dates" [value]="n">{{ n | number:'2.0-0' }}</option>
               </select>
+              <select formControlName="month">
+                <option *ngFor="let n of months" [value]="n">{{ monthsStringArray[n] }}</option>
```

Comment:
I chose to use a simple array to display the months in text form, so that I could essentially leave the rest of the input the same and wouldn't need to do much to change the way job sch are being added